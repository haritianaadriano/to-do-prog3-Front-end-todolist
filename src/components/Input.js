export default function Input({input, setInput}){
    return (
        <div>
            <input 
                data-testid="input-id"
                value={input} 
                placeholder="Enter a task ..."
                onChange={e => setInput(e.target.value)}
            />
        </div>
    )
}