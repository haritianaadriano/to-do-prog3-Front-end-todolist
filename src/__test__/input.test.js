import {render, cleanup, screen} from "@testing-library/react";
import Input from "../components/Input";

test("", ()=>{
    render(<Input/>);
    const inputElement = screen.getByTestId("input-id");
    expect(inputElement).toBeInTheDocument();
})