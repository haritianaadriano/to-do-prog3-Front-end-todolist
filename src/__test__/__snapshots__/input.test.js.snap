// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input test [snapshot] Input renders correctly 1`] = `
<div>
  <input
    data-testid="input-id"
    onChange={[Function]}
    placeholder="Enter a task ..."
  />
</div>
`;
